### ISSUE TITLE
Formatting issue with image manipulation tools menu

### ISSUE THREAD

**Issue by TBRC-pmench** [buda-base/public-digital-library#178]
![Image 2020-07-23 at 2 32 46 PM](https://user-images.githubusercontent.com/48484924/88325880-7cbb7280-ccf3-11ea-8bec-707c7eea877a.png)

**Comment by berger-n** [comment]
I can't reproduce the bug at the time of writing:
![image](https://user-images.githubusercontent.com/33383857/88399592-67177d00-cdc7-11ea-9c5a-487fb3dd1b6e.png)

but I remember seeing it before
my guess is that when there are lots of images/data being requested at once (like when user scrolls a lot, and fast) icons won't be loaded until requested images have been received

it would help to have a way to reproduce it
do you think you can describe a precise succession of steps to observe it?

**Comment by TBRC-pmench** [comment]
I recorded the steps leading to the icon issue. There's no special behavior like scrolling fast. Chrome Version 84.0.4147.89 in Incognito to try to avoid interference from any extensions I have installed.
![Screen Recording 2020-07-24 at 10 51 AM](https://user-images.githubusercontent.com/48484924/88405132-d2982500-cd9c-11ea-8016-c7fdfab5df7e.gif)

![Image 2020-07-24 at 10 48 40 AM](https://user-images.githubusercontent.com/48484924/88403978-433e4200-cd9b-11ea-885b-0fda7c57eddb.png)
